### DISCUSSION TITLE
axis lines seen on object

### DISCUSSION THREAD

**Discussion by Pankaj003** [google/filament#6042] [Q&A]
Hi,

I am seeing dotted axis lines on the 3D object after rendering. 
These axis lines are not there on the source object and seems to be generating while rendering.
Any idea from where these axis lines are coming, how they can be removed? Is there any setting in filament to control them.
I am using filament v.1.20.1

![AxisLine](https://user-images.githubusercontent.com/8086656/188613408-96bd1c53-cc01-4485-8936-5f74616cf54d.PNG)

**Comment by bejado** [comment]
Filament does not have any settings to show axis lines.. they're either an issue with your model or textures.
